pkg/metrics/operator_prom_metrics.go

```diff
@@ -34,7 +34,7 @@ var (
 	crdTotalsGaugeVec = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
 			Namespace: "keda_operator",
-			Subsystem: "cr",
+			Subsystem: "customresource",
```

Comment:
Sorry because I'm late and should have posted it before, but following the other styles, shouldn't this be `custom_recource`?

Reply:
I went with the suggestion Tom made on the docs PR. This is a stylistic issue, and I am not opinionated on it, so I can change it to whichever is preferred.

Reply:
Does it even matter that the resource is custom? From the operator perspective all the resources it owns are custom so I'd just go with `resource`